applications/vila_live/vila_live.yaml

```
@@ -16,10 +16,12 @@
 ---
 v4l2_source:
   device: "/dev/video0"
+  # please consider `pixel_format: "YUYV"` if facing
+  # RuntimeError: Unsupported pixel format NV12
```

@wyli  sorry I wasn't clear, could we please keep the `pixel_format: "YUYV"` that works out of the box for you and add a note that "auto" is also available? Something like:

```suggestion
  pixel_format: "YUYV"    # please update to "auto" if your webcam does not support YUYV
```